Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -296,6 +292,10 @@ extension ProtocolDeclSyntax: EntityNode {
         return inheritanceClause?.types ?? []
     }
 
+    var genericWhereConstraints: [String] {
```

Comment:
[IMHO]

When I initially saw this variable, I couldn't clearly understand what value would return.

To improve readability, how about making sort of GenericWhereConstraints class like the following?

```swift

// where P: C
struct GenericsWhereConstraint {
    let left: String // P
    let right: String // C
}
```

Reply:
It would be a re-implementing `GenericRequirementSyntax`, so I think it goes against the role of the `Models` to convert SwiftSyntax nodes into a struct that is easy to use for code generation.